docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md

```diff
@@ -0,0 +1,211 @@
+---
+author:
+  name: Andrew Leap
+  email: andyleap@gmail.com
+description: 'Alpine Linux is a small, security oriented linux distro.  This guide details installing and configuring Alpine Linux on a KVM Linode'
+keywords: 'alpine,custom,finnix,lish,kvm'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Thursday, March 31st, 2016'
+modified: Thursday, March 31st, 2016
+modified_by:
+  name: Andrew Leap
+title: 'Install Alpine Linux on your Linode'
+contributor:
+  name: Andrew Leap
+  link: [github.com/andyleap](http://github.com/andyleap)
+  external_resources:
+- '[Alpine Linux](http://www.alpinelinux.org/)'
+---
+
+*This is a Linode Community guide. Write for us and earn $250 per published guide.*
+<hr>
+
+[Alpine Linux](http://www.alpinelinux.org/) is a small, security oriented linux distro.
+It is regularly updated with security patches, and runs on the grsec kernel.  All binaries are staticly linked and built against musl libc.
+
+
+## Before You Begin
+
+1.  Familiarize yourself with Lish, as most of this guide will use Lish.
+
+2.  Understand that installing Alpine in this manner will destroy all existing data on the drives you install it on.  Backup all your data first.
+
+3.  The initial portion of this guide will involve creating the drives, so you may need to delete existing drives to have enough free space.
+
+## Setting up the Linode
+
+### Drives
+
+1.  Logon to the (Linode Manager)[https://manager.linode.com/linodes/] and select the linode you will be installing Alpine Linux on.
+
+2.  Create your boot drive, this should be 128~256mb, and the type should be ext4
+
+3.  Create your root drive, with as much space as you need, though if you want a swap partition, make sure you leave room for it.  This drive should be ext4 as well.
+
+4.  (Optionally) Create your swap partition with type swap. (512mb is a good amount of swap)
+
+### Configuration
+
+1. Create a new configuration for your Linode.  Alpine will need to be booted via the Grub 2 kernel, with all the helpers turned off, and boot, root, and swap assigned to /dev/sda, /dev/sdb, and /dev/sdc.
+[![Linode Config](/docs/assets/install-alpine-linux-config_small.png)](/docs/assets/install-alpine-linux-config.png)
+
+## Initial Install of Alpine
+
+### Boot into Recovery
+
+1.  From the Linode Manager, reboot your linode into recovery mode, with your boot drive as /dev/sda, your root drive as /dev/sdb, and your swap as /dev/sdc.
+
+2.  Using Lish, connect to the linode.  (If you are not familiar with Lish, there is a simple web interface for it located under Remote Access)
+
+### Mount Drives
+
+1.  Create a mount point for the root drive
+        mkdir /alpine
+
+2.  Mount and enter the root partition
+        mount /dev/sdb /alpine
+        cd /alpine
+
+3.  Create and mount the boot directory
```

Comment:
I personally don't like the idea of having a cd at the start of every step.  I did add a blurb in the "before you begin" about the fact that a consistent pwd is assumed.